### PR TITLE
Added 3 questions for python, improved distractor answers for previous questions

### PR DESCRIPTION
**Pull request by thegeekyb0y**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

There were some useless distractors in previous questions, I changed them to look a bit thinkable to people.

### CODE REVIEW

**Comment by jdwilkin4** [comment]
> There were some useless distractors in previous questions,

It is important to understand the context for this quiz site.
This quiz site was created as a companion site for the [Learn to Code RPG game](https://www.freecodecamp.org/news/learn-to-code-rpg/). The general audience for that game is for people new to technology and coding.  All of the quiz questions we are creating here will be used in the game too. 

There are some questions that purposefully have "obviously wrong answers" because we do want to have questions that are accessible to that target audience. It is ok if some of the distractors are obviously wrong because that is by design to give people some easy wins.  Especially since we do have some questions that are more intermediate and advanced to satisfy people with more programming and technology knowledge and experience.

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -947,6 +947,33 @@ const pythonQuiz = [
...
     Link: "https://www.freecodecamp.org/news/build-your-first-python-package/",
   },
+  {
+    Question: "What do partition() function do?",
+    Answer: "Splits the string at the first occurence and returns a tuple",
+    Distractor1: "Splits the string at the first occurence and returns a list",
+    Distractor2: "Splits the string equally",
+    Distractor3: "Split the string at any occurence of the given argument",
+    Explanation: "The partition() method searches for a specified string, and splits the string into a tuple containing three elements.",
+    Link: "https://www.w3schools.com/python/ref_string_partition.asp",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:957]: Please use either a freeCodeCamp resource like an article or youtube video or use official python documentation. 

https://docs.python.org/3/library/stdtypes.html#str.partition

File: src/data/python-quiz.ts [hunk 2]
```diff
@@ -947,6 +947,33 @@ const pythonQuiz = [
     Explanation: "We cannot create a Python package without adding a __init__.py file, This file will tell Python to treat directories as modules.",
     Link: "https://www.freecodecamp.org/news/build-your-first-python-package/",
   },
+  {
+    Question: "What do partition() function do?",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:951]:
```suggestion
    Question: "What does the partition() function do?",
```

**Comment by thegeekyb0y** [comment]
Oh okay, should I reverse those commits?

File: src/data/python-quiz.ts [hunk 3]
```diff
@@ -947,6 +947,33 @@ const pythonQuiz = [
...
+    Answer: "Splits the string at the first occurence and returns a tuple",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:952]: spelling error here

```suggestion
    Answer: "Splits the string at the first occurrence and returns a tuple",
```

File: src/data/python-quiz.ts [hunk 4]
```diff
@@ -947,6 +947,33 @@ const pythonQuiz = [
...
+    Distractor1: "Splits the string at the first occurence and returns a list",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:953]:
```suggestion
    Distractor1: "Splits the string at the first occurrence and returns a list",
```

File: src/data/python-quiz.ts [hunk 5]
```diff
@@ -947,6 +947,33 @@ const pythonQuiz = [
...
+    Distractor2: "Splits the string equally",
+    Distractor3: "Split the string at any occurence of the given argument",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:955]:
```suggestion
    Distractor3: "Split the string at any occurrence of the given argument",
```

File: src/data/python-quiz.ts [hunk 6]
```diff
@@ -947,6 +947,33 @@ const pythonQuiz = [
...
+    Link: "https://www.w3schools.com/python/ref_string_partition.asp",
+  },
+  {
+    Question: "An empty/null statement in Python is",
+    Answer: "pass",
+    Distractor1: "continue",
+    Distractor2: "over",
+    Distractor3: "continue",
+    Explanation: "An empty statement is a statement that do nothing. Python offers pass statement as empty statement",
+    Link: "https://www.geeksforgeeks.org/how-to-write-an-empty-function-in-python-pass-statement/",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:966]: Please use either a freeCodeCamp resource like an article or youtube video or use official python documentation.

The docs would be good in this case
https://docs.python.org/3/tutorial/controlflow.html#pass-statements

File: src/data/python-quiz.ts [hunk 7]
```diff
@@ -947,6 +947,33 @@ const pythonQuiz = [
...
+    Explanation: "The partition() method searches for a specified string, and splits the string into a tuple containing three elements.",
+    Link: "https://www.w3schools.com/python/ref_string_partition.asp",
+  },
+  {
+    Question: "An empty/null statement in Python is",
+    Answer: "pass",
+    Distractor1: "continue",
+    Distractor2: "over",
+    Distractor3: "continue",
+    Explanation: "An empty statement is a statement that do nothing. Python offers pass statement as empty statement",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:965]:
```suggestion
    Explanation: "An empty statement is a statement that does nothing. Python offers the pass statement as an empty statement.",
```

File: src/data/python-quiz.ts [hunk 8]
```diff
@@ -947,6 +947,33 @@ const pythonQuiz = [
...
+    Question: "What do partition() function do?",
+    Answer: "Splits the string at the first occurence and returns a tuple",
+    Distractor1: "Splits the string at the first occurence and returns a list",
+    Distractor2: "Splits the string equally",
+    Distractor3: "Split the string at any occurence of the given argument",
+    Explanation: "The partition() method searches for a specified string, and splits the string into a tuple containing three elements.",
+    Link: "https://www.w3schools.com/python/ref_string_partition.asp",
+  },
+  {
+    Question: "An empty/null statement in Python is",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:960]:
```suggestion
    Question: "What is an empty/null statement in Python called?",
```

File: src/data/python-quiz.ts [hunk 9]
```diff
@@ -947,6 +947,33 @@ const pythonQuiz = [
...
+    Answer: "pass",
+    Distractor1: "continue",
+    Distractor2: "over",
+    Distractor3: "continue",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:964]: You have continue twice.
Let's pick something else

File: src/data/python-quiz.ts [hunk 10]
```diff
@@ -947,6 +947,33 @@ const pythonQuiz = [
...
+    Explanation: "An empty statement is a statement that do nothing. Python offers pass statement as empty statement",
+    Link: "https://www.geeksforgeeks.org/how-to-write-an-empty-function-in-python-pass-statement/",
+  },
+  {
+    Question: "Function range(4) will yeild an iterable sequence like: ",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:969]:
```suggestion
    Question: "In Python, Function range(4) will yield an iterable sequence like: ",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by thegeekyb0y** [comment]
All requested changes applied.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**